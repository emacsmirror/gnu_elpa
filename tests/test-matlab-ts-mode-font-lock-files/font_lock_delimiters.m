% -*- matlab-ts -*-

foo.bar = 1;

a = (1, 2);

x = 1:10;


