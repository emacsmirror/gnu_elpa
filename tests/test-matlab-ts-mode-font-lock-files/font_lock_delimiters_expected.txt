c ccc ccccccccc ccc

dddDddd o nD

v o bnD nwD

v o nDnnD


